handbook/business-operations/business-operations.rituals.yml

```diff
@@ -52,114 +22,169 @@
-  dri: "joStableford" 
+  dri: "jostableford" 
   autoIssue:   
     labels: [ "#g-business-operations" ] 
     repo: "confidential" 
 -
-  task: "Run payroll" 
-  startedOn: "2024-02-28" 
-  frequency: "Monthly" 
-  description: "Check Gusto and Plane manually for accuracy" 
-  moreInfoUrl: "https://fleetdm.com/handbook/business-operations#inform-managers-about-hours-worked"
-  dri: "joStableford" 
+  task: "Run US commission payroll" 
+  startedOn: "2024-01-31" 
+  frequency: "Monthly"
+  description: "Verify closed-won deal amounts, use commission calculators to determine commissions owed, and process payroll." 
+  moreInfoUrl: "https://fleetdm.com/handbook/business-operations#run-us-commission-payroll"
+  dri: "jostableford" 
   autoIssue:   
     labels: [ "#g-business-operations" ] 
-    repo: "confidential" 
+    repo: "confidential"
+-
+  task: "Run bonus payroll" 
+  startedOn: "2024-01-31" 
+  frequency: "Quarterly"
+  description: "Verify completion of any objective or outcome based bonus plans, and process payroll." 
+  moreInfoUrl: "https://fleetdm.com/handbook/business-operations#run-us-commission-payroll" # TODO update linked process and add a new process that captures MBO payment
+  dri: "jostableford" 
+  autoIssue:   
+    labels: [ "#g-business-operations" ] 
+    repo: "confidential"
 -
   task: "Investor reporting" 
   startedOn: "2024-03-31" 
   frequency: "Quarterly"
-  description: "Provide updated metrics for CRV in Chronograph" 
+  description: "Provide updated metrics for CRV in Chronograph." 
   moreInfoUrl: "https://fleetdm.com/handbook/business-operations#report-quarterly-numbers-in-chronograph"
   dri: "hollidayn"
+  autoIssue:   
+    labels: [ "#g-business-operations" ] 
+    repo: "confidential"
 -
   task: "Quartlery finance check" 
   startedOn: "2024-03-31" 
   frequency: "Quarterly"
-  description: "Create the quarterly close GitHub issue and walk through the steps" 
+  description: "Every quarter, we check Quickbooks Online (QBO) for discrepancies and follow up with accounting providers for any quirks found." 
   moreInfoUrl: "https://fleetdm.com/handbook/business-operations#check-finances-for-quirks"
-  dri: "joStableford"
+  dri: "jostableford"
+  autoIssue:   
+    labels: [ "#g-business-operations" ] 
+    repo: "confidential"
 -
   task: "Quarterly grants" 
-  startedOn: "2024-03-31" 
+  startedOn: "2024-02-01" 
   frequency: "Quarterly"
-  description: "Create the quarterly close GitHub issue and walk through the steps" 
+  description: "Create the equity grants GitHub issue and walk through the steps." 
   moreInfoUrl: "https://fleetdm.com/handbook/business-operations#grant-equity"
   dri: "hollidayn" 
+  autoIssue:   
+    labels: [ "#g-business-operations" ] 
+    repo: "confidential"
 -
   task: "Deliver annual report for venture line" 
-  startedOn: "2024-11-01" 
+  startedOn: "2024-12-01" 
   frequency: "Annually"
```

Comment:
Hey @JoStableford, unfortunately, we don't support "Annually" or "Quarterly" as a frequency for auto issues (I do have an issue for it though). That's why this PR is still failing test fleet website.